doc/question.md

```diff
@@ -97,14 +101,29 @@ The `question.html` is a template used to render the question to the student. A
 <p>$F = $ <pl-number-input answers_name="F" comparison="sigfig" digits="2" /> $\rm m/s^2$
 ```
 
-The `question.html` is regular HTML, with three special features:
+The `question.html` is regular HTML, with four special features:
 
 1. Any text in double-curly-braces (like `{{params.m}}`) is substituted with variable values. This is using [Mustache](https://mustache.github.io/mustache.5.html) templating.
 
 2. Special HTML elements (like `<pl-number-input>`) enable input and formatted output. See the [list of PrairieLearn elements](elements.md).
    
 3. A special `<markdown>` tag allows you to write Markdown inline in questions.
 
+4. LaTeX equations are available by putting `$y = x^2$` or `\[ y = \int_0^1 x^2\ dx \]` into the HTML.
+
+## The `singleVariant` option for non-randomized questions
+
+While it is recommended that all questions contain random parameters, sometimes it is impractical to do this. For questions that don't have a meaningful amount of randomization in them, the `info.json` file should set `"singleVariant": true`. This has the following effects:
+
+* On Homeworks, each student will only ever be given one variant of the question, which they can repeatedly attempt without limit. The correct answer will never be shown to students.
+* On Exams, the `singleVariant` option has no effect and the question is treated like any other.
+
+## The `partialCredit` option
+
+By default all questions award partial credit. For example, if there are two numeric answers in a question and only one of them is correct then the student will be awarded 50% of the available points.
+
+To disable partial credit for a question, set `"partialCredit": false` in the `info.json` file for the question. This will mean that the question will either give 0% or 100%, and it will only give 100% if every element on the page is fully correct.
```

Comment:
Maybe mention that some elements (e.g. `pl-checkbox`) have more fine grain control over partial credit being awarded?

Reply:
done